README.md

```diff
@@ -12,7 +20,7 @@ This project uses the general contract message relaying script/predicate from th
 
 | dep     | version                                                  |
 | ------- | -------------------------------------------------------- |
-| Forc    | [v0.30.0](https://fuellabs.github.io/sway/v0.30.0/introduction/installation.html) |
+| Forc    | [v0.31.1](https://fuellabs.github.io/sway/v0.31.1/introduction/installation.html) |
```

Comment:
I would omit the `v` in the table since the heading is literally `version`

Reply:
I personally don't mind it since that's how it's referenced in the URL and how it's tagged on GitHub, and other places. It comes from how we've been doing it to reference nodejs dependencies in our Solidity repos